src/core/reducers/session.ts

```diff
@@ -18,6 +18,12 @@ export default function updateSession(state: State = {}, action: Action): State
         if ('searchId' in action.meta) {
```

Comment:
these can all just use a function that's something like `updateSection` where it updates the key given